libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -45,4 +107,39 @@ export class SkyLabelComponent implements OnInit {
     this.baseIcon = indicatorIcon.modernThemeBaseIcon;
     this.topIcon = indicatorIcon.modernThemeTopIcon;
   }
+
+  #updateDescriptionComputed(): void {
+    this.#unsubscribe();
```

Comment:
Can we be a little more descriptive on this method and the name of the `currentSub`?

Could we rename to something like `descriptionTypeResourceSubscription` and `descriptionTypeResourceUnsubscribe`? Maybe doesn't have to be that wordy - but something to say what these are in case we add another subscription or similar in the future.